### PR TITLE
Tests for ReduceLogSum

### PR DESCRIPTION
**Pull request by anirudhacharya**
- [x] Add Model and data files.

### CODE REVIEW

File: onnx/backend/test/case/node/reduce_log_sum.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+class ReduceLogSum(Base):
+
+    @staticmethod
+    def export():
```
Comment (houseroad) [inline onnx/backend/test/case/node/reduce_log_sum.py]: Nit: change the function to export_nokeepdims

File: onnx/backend/test/case/node/reduce_log_sum.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                axes=[2, 1],
+                keepdims=0
+            )
+        data = np.random.ranf([3, 4, 5]).astype("float32")
+        reduced = np.log(np.sum(data, axis=(2, 1), keepdims=False))
+        expect(node, inputs=[data], outputs=[reduced],
+                   name='test_reduce_log_sum')
+
+    @staticmethod
+    def export_default():
```
Comment (houseroad) [inline onnx/backend/test/case/node/reduce_log_sum.py]: Nit: change the function to export_nokeepdims

File: onnx/backend/test/case/node/reduce_log_sum.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    @staticmethod
+    def export():
+        node = onnx.helper.make_node(
+                'ReduceLogSum',
+                inputs=['data'],
+                outputs=["reduced"],
+                axes=[2, 1],
+                keepdims=0
+            )
+        data = np.random.ranf([3, 4, 5]).astype("float32")
```
Comment (houseroad) [inline onnx/backend/test/case/node/reduce_log_sum.py]: to be consistent, better using astype(np.float32)

File: onnx/backend/test/case/node/reduce_log_sum.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        reduced = np.log(np.sum(data, axis=(2, 1), keepdims=False))
```
Comment (houseroad) [inline onnx/backend/test/case/node/reduce_log_sum.py]: right now, both cases are descending order, we can have one case with ascending axes.

**Comment by houseroad** [comment]
Please call https://github.com/onnx/onnx/blob/master/onnx/defs/gen_doc.py to update the generated docs.

**Comment by houseroad** [comment]
Please lint the code accordingly. https://travis-ci.org/onnx/onnx/jobs/374574385#L7310

**Comment by anirudhacharya** [comment]
@houseroad please merge this if I have addressed all your comments.

**Pull request merged**